src/transformers/trainer_seq2seq.py

```diff
@@ -39,6 +41,9 @@
     from .trainer_utils import EvalPrediction, PredictionOutput
     from .training_args import TrainingArguments
 
+    if is_datasets_available():
+        import datasets
```

Comment:
Thanks! The quality check is for this chunk here. I think it'd be fine if this was outside `TYPE_CHECKING`

Reply:
Thank you 🙌